integr8ly/installation
# Intly 2615

Adds a dashboard to display the resources per namespace.

Verification steps:

1. Run the installer from this branch
1. Open Grafana
1. Make sure there are the following dasboards: 'Resource Usage By Middleware Service', 'Resources by Namespace' and 'Cluster Resources'
1. Make sure that the namespace dropdown in the dashboards contains only middleware services
1. Make sure that all dashboards display data and that namespace / pod data is limited to middleware namespaces.
